cmd/kustomizepatch/config.go

```diff
@@ -0,0 +1,58 @@
+package kustomizepatch
+
+import "github.com/giantswarm/microerror"
+
+// config is required configuration passed by the generator
+// Example:
+// ```
+// apiVersion: generators.giantswarm.io/v1
+// kind: Konfigure
+// metadata:
+//   name: konfigure-plugin
+//   annotations:
+//     config.kubernetes.io/function: |
+//       exec:
+//         path: /plugins/konfigure
+// app_catalog: ""
+// app_destination_namespace: ""
+// app_disable_force_upgrade: true
+// app_name: ""
+// app_version: ""
+// name: ""
+// ```
+type config struct {
```

Comment:
This file is very similar to `cmd/generate/flag.go`. Some of the flags did not apply to kustomize plugins. Also, the whole `config` struct will be supplied by `kustomize`, so we don't have to worry about command-line interfaces.